canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -570,13 +570,22 @@ class PropertiesMain extends React.Component {
 					{editorForm}
 				</PropertiesModal>);
 			}
+
+			// If Custom width is set adjust classname to format action buttons
+			const customWidth = this.propertiesController.getForm().pixelWidth;
```

Comment:
@srikant-ch5 is there an example where I can see this working?  Can't you just set the class based on `overrideSize` being set?  

```
			if (overrideSize !== null) {
				overrideStyle = { width: overrideSize + "px" };
			}
```

Reply:
Hi @matthoward366 If I include it under `overrideSize` condition below test conditions are failing.
- When enableResize=true and editor_size=small and pixel_width min and max are set resize button should be rendered
- When enableResize=true and editor_size=small and pixel_width min and max are the same the resize button should not be rendered
- When enableResize=true and editor_size is omitted and pixel_width min and max are the same the resize button should not be rendered
https://github.com/elyra-ai/canvas/actions/runs/10934918203/job/30355802106?pr=2150

Reply:
Any reason not to update the tests?  In all the above cases the width is going to by the `overrideSize` and not use the className to set the size.

Reply:
I have updated tests and included classname in overrideSize condition.